user_docs/en/userGuide.t2t

```diff
@@ -66,9 +66,10 @@ For further details, you can [view the full license. https://www.gnu.org/license
 For details regarding exceptions, access the license document from the NVDA menu under the "help" section.
 
 + System Requirements +[SystemRequirements]
-- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, and all Server Operating Systems starting from Windows Server 2008 R2.
+- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 11 (64-bit only), and all Server Operating Systems starting from Windows Server 2008 R2.

```

Comment:
I have a concern that specifying this might confuse some readers, as the following is still a true statement:

```all 32-bit and 64-bit editions of Windows 7, ..., Windows 11, and all .. ```